src/test/java/plugins/AntPluginTest.java

```diff
@@ -23,15 +32,74 @@ public class AntPluginTest extends AbstractJUnitTest {
     private static final String INSTALL_VERSION_1_10 = "1.10.5";
     private static final String INSTALL_NAME_1_10 = "ant_" + INSTALL_VERSION_1_10;
 
+    private static final String NATIVE_ANT_NAME = "native_ant";
+
+    private static final String OK_PROP1 = "okPROP1=foo_bar_ok_1";
+    private static final String OK_PROP2 = "okPROP2=foo_bar_ok_2";
+    private static final String NOK_PROP1 = "nokPROP1=foo_bar_nok_1";
+    private static final String NOK_PROP2 = "nokPROP2=foo_bar_nok_2";
+    private static final String PROPERTIES = OK_PROP1+"\n"+OK_PROP2+"\n"+NOK_PROP1+"\n"+NOK_PROP2;
```

Comment:
as these appear to be specific to a single test would they be better off defined in that single test method?